examples/player/simid_player.js

```diff
@@ -245,67 +260,145 @@ class SimidPlayer {
   }
 
   /**
-   * Returns the original specified dimensions of the non-linear creative.
-   * @return {!Object}
+   * Checks whether the input dimensions are valid and fit in the player window.
+   * @param {!Object} dimensions A dimension that contains x, y, width & height fields.
```

Comment:
Looks like we pass an object that specifically has {x, y, width, height} through a few of these functions .. it might be good to formalize this as a class or a record type at least (can push this to a different pr, don't need to do as part of this one)